components/atom/input/src/Input/Features/Icon/index.scss

```diff
@@ -0,0 +1,53 @@
+$l-atom-input-icon: $gutter !default;
+$r-atom-input-icon: $gutter !default;
+$w-atom-input-icon: $sz-icon-m !default;
+$h-atom-input-icon: $sz-icon-m !default;
+$t-atom-input-icon: 50% !default;
+$trf-ty-atom-input-icon: -50% !default;
+$pl-atom-input-icon: 50px !default;
+$pl-atom-input-input: $w-atom-input-icon + $l-atom-input-icon + $p-s !default;
+$pr-atom-input-input: $w-atom-input-icon + $r-atom-input-icon + $p-s !default;
+
+$base-class: '.sui-AtomInput';
+$class-input: '#{$base-class}-input';
+
+#{$base-class} {
+  &--withIcon {
+    position: relative;
+    width: 100%;
+
+    &--left #{$class-input} {
+      padding-left: $pl-atom-input-input;
+    }
+
+    &--right #{$class-input} {
+      padding-right: $pr-atom-input-input;
+    }
+
+    &-icon {
+      align-items: center;
+      color: $c-primary;
+      cursor: pointer;
```

Comment:
maybe this should only happen if there is an onClick action, should be easy adding an extra css class in case the onClick is defined